apps/studio/components/interfaces/Storage/VectorBuckets/VectorsBuckets.tsx

```diff
@@ -150,6 +155,24 @@ export const VectorsBuckets = () => {
           )}
         </ScaffoldSection>
       )}
+
+      <DeleteBucketModal
+        visible={modal === 'delete'}
+        bucket={{
+          id: selectedBucket?.vectorBucketName || '',
+          name: selectedBucket?.vectorBucketName || '',
+          created_at: selectedBucket?.creationTime || '',
+          updated_at: selectedBucket?.creationTime || '',
+          owner: '',
+          public: false,
+          type: 'STANDARD' as const,
```

Comment:
I had to add these values to keep `DeleteBucketModal` happy. Not sure why, as analytics buckets work fine as-is. See https://github.com/supabase/supabase/pull/39610